tools/win32/detect_cpu_extensions_wine.py

```diff
@@ -68,14 +69,12 @@ def path_as_windows(fpath):
 
 def write_summary(allcodes):
     """Write a summary of all found codes to stdout."""
-    print """\n
-----------------------------------------------------------------------------
-Checked all binary files for CPU extension codes. Found the following codes:"""
+    print("\n"
+          "----------------------------------------------------------------------------"
+          "Checked all binary files for CPU extension codes. Found the following codes:")
```

Comment:
I'm surprised we're checking these, maybe it's easier to just skip checking the docstrings of anything under tools/ ? 

Also this code in particular might be dead code --- we've dropped building under wine around scipy 0.17.0 IIRC.